package.json

```
@@ -19,6 +19,7 @@
     "build:debug-memory": "next build --experimental-debug-memory-usage",
     "start": "next start",
     "lint": "next lint",
+    "lint:types": "npx tsc --noEmit",
```

```suggestion
    "lint:types": "tsc --noEmit",
```

Is enough. `tsc` is in binaries